### DISCUSSION TITLE
"recently played" should also sort difficulties by the same parameter

### DISCUSSION THREAD

**Discussion by marioman63** [ppy/osu#26109] [Ideas] [answered]
When sorting by "recently played", songs are listed in the order last played, with the most recent at the top. However, individual maps/difficulties in each song are still the default sorting (easiest to hardest). It would be nice if the order of difficulties reflected the order they were last played (with most recent difficulty at the top of the list)

**Comment by peppy** [comment, marked as answer]
We are storing last play information per beatmap, so there's a high change this will be possible once song select supports splitting up beatmaps in a better way.
